.github/workflows/rust.yml

```diff
@@ -18,25 +18,59 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        redis:
-          - 6.2.13
-          - 7.2.0
-        rust:
-          - stable
-          - beta
-          - nightly
-          - 1.65.0
+        config:
+          [
+            # Different DB cases:
+            {
+              rust: stable,
+              db-org: redis,
+              db-name: redis,
+              db-version: 6.2.13
+            },
+            {
+              rust: stable,
+              db-org: redis,
+              db-name: redis,
+              db-version: 7.2.4
+            },
+            {
+              rust: stable,
+              db-org: valkey-io,
+              db-name: valkey,
+              db-version: 7.2.6
+            },
+
+            # Different rust cases
+            {
+              rust: beta,
+              db-org: redis,
+              db-name: redis,
+              db-version: 7.2.4
+            },
+            {
+              rust: nightly,
+              db-org: redis,
+              db-name: redis,
+              db-version: 7.2.4
+            },
+            {
+              rust: 1.65.0,
+              db-org: redis,
+              db-name: redis,
+              db-version: 7.2.4
```

Comment:
Thanks again!
Can you change the default engine to  valkey 7.2.4?

Reply:
done

Reply:
tests failed because there's no valkey 7.2.4 in that path. returned to 7.2.6